src/Symfony/Component/BrowserKit/HttpBrowser.php

```diff
@@ -97,7 +109,7 @@ private function getHeaders(Request $request): array
                 $headers[substr($key, 5)] = $value;
             } elseif (isset($contentHeaders[$key])) {
                 // CONTENT_* are not prefixed with HTTP_
-                $headers[$key] = $value;
+                $headers[ucwords($key, '-')] = $value;
```

Comment:
Really needed?

Reply:
If I remove it, content type header in the request will be `content-type`,  that is not consistent at least in test cases. I can actually revert it, but I will need to modify the test.